### ISSUE TITLE
Font Awesome ... icon not showing up for user

### ISSUE THREAD

**Issue by jSylvestre** [ucdavis/Peaks#1246]
![image](https://user-images.githubusercontent.com/469010/223193125-cbf4b53a-41ca-491f-8377-6e70726d6672.png)

**Comment by jSylvestre** [comment]
Another user having this issue
INC1623026

**Comment by jSylvestre** [comment]
If I block remote fonts, I still see a square, so it is something different?

![image](https://user-images.githubusercontent.com/469010/236548434-44160a20-6097-4d08-b2c3-facebb0fee8c.png)

**Comment by jcubbage** [comment]
Yes. FA works by using CSS to inject glyph. So, I'm thinking they are blocking the FA CDN or having another problem with that loading. Also...on your test, did you clear cache? I _think_ if  you block a CDN but already have the files, the browser will use the cached copy?

**Comment by jSylvestre** [comment]
Not sure, second person never got back to me with anything meaningful.
Just blocked fonts with ublockOrigin.

**Comment by srkirkland** [comment]
@laholstege Start by updating all FA package.json libs to current versions.  Check out https://fontawesome.com/docs/web/use-with/react/
